Gemfile

```diff
@@ -40,7 +40,7 @@ gem 'puma'
 gem 'rack-cors'
 
 # Redis
-gem 'redis-rails'
+gem 'redis'
```

Comment:
I had to include the Redis gem explicitly.
I guess that was not necessary in the BE because it is using the `redis-elasticache` which requires Redis.